test/models/project_media_8_test.rb

```diff
@@ -0,0 +1,72 @@
+require_relative '../test_helper'
+
+class ProjectMedia8Test < ActiveSupport::TestCase
+  def setup
+    super
+    require 'sidekiq/testing'
+  end
+
+  test ":create_tags should create tags when project media id and tags are present" do
+    team = create_team
+    project = create_project team: team
+    pm = create_project_media project: project
+
+    assert_nothing_raised do
+      ProjectMedia.create_tags(project_media_id: pm.id, tags_json: ['one', 'two'].to_json)
+    end
+    assert_equal 2, pm.annotations('tag').count
+  end
+
+  test ":create_tags should not raise an error when no project media is sent" do
+    assert_nothing_raised do
+      CheckSentry.expects(:notify).once
+      ProjectMedia.create_tags(project_media_id: nil, tags_json: ['one', 'two'].to_json)
+    end
+  end
+
+  test "when creating an item with tag/tags, after_create :create_tags_in_background callback should create tags in the background" do
+    Sidekiq::Testing.inline!
+
+    team = create_team
+    project = create_project team: team
+    pm = create_project_media project: project, tags: ['one']
```

Comment:
The setter `tags` works? I thought it was `set_tags`.

Reply:
@caiosba this `create_project_media` is a method from `sample_data`, here is the relevant line:

https://github.com/meedan/check-api/blob/dffb66d1040398aecf78e43cec8be0840605a04e/lib/sample_data.rb#L520